examples/StorageAccount/StorageAccount_Example.md

```diff
@@ -0,0 +1,53 @@
+# How To Run StorageAccount.Tests.ps1
+
+`StorageAccount.Tests.ps1` contains examples of using the `Confirm-AzBPStorageAccount` cmdlet.
+
+## Pre-Requisites
```

Comment:
Update this doc to reuse the setup instructions at the root of the examples folder. Please reference main branch for how other docs are doing this.

Reply:
I don't see a difference between the Pre-Requisites and Common Steps sections in this file and Readme in the root of the examples folder. can you be more specific, please?